sdk/storage/azure-storage-blobs/src/blob_client.cpp

```diff
@@ -531,6 +532,38 @@ namespace Azure { namespace Storage { namespace Blobs {
         *m_pipeline, m_blobUrl, protocolLayerOptions, context);
   }
 
+  Azure::Response<Models::CopyBlobFromUriResult> BlobClient::CopyFromUri(
+      const std::string& sourceUri,
+      const CopyBlobFromUriOptions& options,
+      const Azure::Core::Context& context) const
+  {
+    _detail::BlobRestClient::Blob::CopyBlobFromUriOptions protocolLayerOptions;
+    protocolLayerOptions.Metadata = options.Metadata;
+    protocolLayerOptions.Tags = options.Tags;
+    protocolLayerOptions.SourceUri = sourceUri;
+    protocolLayerOptions.AccessTier = options.AccessTier;
```

Comment:
Official doc does not allow access tier. Could you please double-check/target test whether it is supported?
https://docs.microsoft.com/en-us/rest/api/storageservices/copy-blob-from-url

Reply:
It's supported according to swagger https://github.com/Azure/azure-rest-api-specs/blob/2fbdd238b9fc19884e2d0491a338f7f6fc2aa1d7/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-02-10/blob.json#L6123

Reply:
and it's covered by unit test